Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v6.1.0
+- Add support of web.app as an auth domain. (#2959) 
```

Comment:
This line has a trailing whitespace, please fix to make Travis pass.

Reply:
Done.